tests/contract/test_resource_client.py

```diff
@@ -1238,7 +1248,24 @@ def test_transform_model_equal_output(resource_client):
     output_model = TRANSFORM_OUTPUT.copy()
 
     resource_client.transform_model(input_model, output_model)
-    assert input_model == TRANSFORM_OUTPUT
+    assert input_model == output_model
+
+
+def test_transform_model_equal_output_OR(resource_client):
+    input_model = {"e": "new", "a": "ValueA", "c": 1}
+    output_model = {"e": "newValue", "a": "ValueA", "c": 1}
+
+    resource_client.transform_model(input_model, output_model)
+    assert input_model == output_model
+
+
+def test_transform_model_not_equal_output_OR(resource_client):
```

Comment:
It suprised me a little bit that we're testing that `"c": 5` cause a not equal. I would have expected the test to be focussed around `e` not being equal to the input. E.g. with `"e": "not-newValue"`

Reply:
makes sense. I will update the test to test the OR fail condition.